src/lib/objects.js

```diff
@@ -0,0 +1,87 @@
+const createPerson = (name, age) => {
+  const person = {};
+  person.name = name;
+  person.age = age;
+  return person;
+};
+
+const getName = object => {
+  return object.name;
+};
+
+const getProperty = (property, object) => {
+  if (property === 'age') {
+    return object.age;
+  }
+  return object.name;
+};
+
+const hasProperty = (property, object) => {
+  if (object.hasOwnProperty(property)) {
+    return true;
+  }
+  return false;
+};
+
+const isOver65 = person => {
+  if (person.age > 65) {
+    return true;
+  }
+  return false;
+};
+
+const getAges = people => {
+  function getAges(input, field) {
+    var output = [];
+    for (var i = 0; i < input.length; i++) output.push(input[i][field]);
+    return output;
+  }
+  var result = getAges(people, 'age');
+  return result;
```

Comment:
This could be re-written. First thing to point out is that you are creating a function with the same name as the parent, which is bad practice. Also, you can make use of built in methods like filter and map to the same job for you  a bit more concisely:
```
  return people.filter(person => person.name).map(person => person.age);
```